src/settings/browser-settings.js

```diff
@@ -262,11 +276,24 @@ const Settings = () => {
           </div>
         </div>
       </nav>
+      <nav class="panel">
+        <p class="panel-heading">Global notifications</p>
+        <div class="panel-block">
+          <div class="settings__global-notifications container">
+            <${SettingsButton}
+              icon=${state.notifications ? 'fa-toggle-on' : 'fa-toggle-off'}
+              title="Global control of notifications for ElectronIM. If disabled, overrides any notification enabled for tabs."
+              onclick=${toggleNotifications}
+            />
```

Comment:
Please use a standard checkbox here (Label should be aligned with the [previous comment](https://github.com/manusa/electronim/pull/28/files#r457200235))
e.g. A checkbox labeled "Disable notifications", "Disable notifications globally"..

Reply:
Done